docs/dep/dep-0001.rst

```diff
@@ -0,0 +1,599 @@
+good in principle sometimes turn out to be impractical when subjected to the
+test of implementation.
+
+DEP authors are responsible for collecting community feedback on a DEP before
+submitting it for review. However, wherever possible, long open-ended
+discussions on public mailing lists should be avoided. Strategies to keep the
+discussions efficient include: having the DEP author accept private comments in
+the early design phases, setting up a wiki page, etc. DEP authors should use
+their discretion here.
+
+
+DEP Review & Resolution
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the authors have completed a DEP, they may request a review for style and
+consistency from the DEP editors. However, the content and final acceptance of
+the DEP must be requested of the DebOps maintainers, usually via an email to
+the debops-users mailing list. DEPs are reviewed by the DebOps maintainers and
+their chosen consultants, who may accept or reject a DEP or send it back to the
+author(s) for revision. For a DEP that is predetermined to be acceptable (e.g.,
+it is an obvious win as-is and/or its implementation has already been checked
+in) the DebOps maintainers may also initiate a DEP review, first notifying the
+DEP author(s) and giving them a chance to make revisions.
+
+The final authority for DEP approval are the DebOps maintainers. However,
+whenever a new DEP is put forward, any core developer that believes they are
+suitably experienced to make the final decision on that DEP may offer to serve
+as the DebOps maintainers delegate (or "DEP czar") for that DEP. If their
+self-nomination is accepted by the other core developers and the DebOps
+maintainers, then they will have the authority to approve (or reject) that DEP.
+This process happens most frequently with DEPs where the DebOps maintainers
+have granted in principle approval for *something* to be done, but there are
+details that need to be worked out before the DEP can be accepted.
+
+If the final decision on a DEP is to be made by a delegate rather than directly
+by the DebOps maintainers, this will be recorded by including the
+"DebOps-Delegate" header in the DEP.
+
+DEP review and resolution may also occur on a list other than debops-users. In
+this case, the "Discussions-To" heading in the DEP will identify the
+appropriate alternative list where discussion, review and pronouncement on the
+DEP will occur.
+
+For a DEP to be accepted it must meet certain minimum criteria. It must be
+a clear and complete description of the proposed enhancement. The enhancement
+must represent a net improvement. The proposed implementation, if applicable,
+must be solid and must not complicate the codebase unduly. Finally, a proposed
+enhancement must be "debopsy" in order to be accepted by the DebOps
+maintainers. (However, "debopsy" is an imprecise term; it may be defined as
+whatever is acceptable to the DebOps maintainers. This logic is intentionally
+circular.).
```

Comment:
debopsy :smile: Throwing in a joke here and there is a good idea with such long and otherwise welcoming precise specifications.

Reply:
I cannot take the credit here, it's in the original PEP 1 as well. :-)